### PR TITLE
chore(contactus): styling fixes based on design feedback

### PR DESCRIPTION
**Pull request by dcshzj**
## Problem

Closes IS-538.

## Solution

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
- [x] No - this PR is backwards compatible

**Improvements**:

- Some padding and copy changes were made based on the feedback obtained from @sehyunidaaa.

## Tests

- [ ] Unit tests (using `npm run tests`)
- [ ] e2e tests (comment on this PR with the text `!run e2e`)
- [x] Smoke tests
    - [ ] When creating a new location and contact information card, the default card title should no longer be in titlecase
    - [ ] The bottom padding of the general information section should now be larger
    - [ ] The guiding information underneath the feedback form URL input box should be updated
    - [ ] "Remove operating hours" should be updated to say "Delete operating hours"
    - [ ] There should be more space between the "Add operating hours" and the last operating hours card
    - [ ] The Add section button should now be slightly bigger
    - [ ] The title text in an empty section should now be centre-aligned when the title text spills over to two lines
    - [ ] The extra spacing when scrolling the sidebar all the way to the bottom should now be removed
    - [ ] The size of the accordion of a non-draggable card should now be smaller
    - [ ] The title text of the empty section placeholder now uses `subhead-1` text styling

## Deploy Notes

*None*

### CODE REVIEW

File: src/layouts/components/ContactUs/GeneralInfoSection.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const GeneralInfoSection = ({
       title="General Information"
       isInvalid={!!errors.agency_name || !!errors.feedback}
     >
-      <Editable.Section spacing="1.25rem">
+      <Editable.Section spacing="1.25rem" pb="1.25rem">
```
Comment (seaerchin) [inline src/layouts/components/ContactUs/GeneralInfoSection.tsx:39]: is the `pb` only valid for `GeneralInfoSection`?

File: src/layouts/components/Editable/AddSectionButton.tsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ export const AddSectionButton = ({
             isStretch
             isOpen={isOpen}
             rightIcon={undefined}
+            size="lg"
```
Comment (seaerchin) [inline src/layouts/components/Editable/AddSectionButton.tsx:23]: assumed to be cleared w/ design

File: src/layouts/components/Editable/Editable.tsx [hunk 3]
```diff
@@ -71,7 +71,13 @@ export const EmptySection = ({
...
     >
       {image}
-      <Text>{title}</Text>
+      <Text
+        textColor="base.content.strong"
+        textAlign="center"
+        lineHeight="1.75rem"
+      >
+        {title}
+      </Text>
```
Comment (seaerchin) [inline src/layouts/components/Editable/Editable.tsx:80]: assumed to be cleared with design

File: src/layouts/components/Editable/Editable.tsx [hunk 4]
```diff
@@ -217,7 +222,7 @@ const EditableAccordionItem = ({
           {/* NOTE: Check with design on styling.
         See if entire section is button (ie, whole component hover styling)
       */}
-          <AccordionButton px="1.5rem" py="3rem">
+          <AccordionButton px="1.5rem" py="2.125rem">
```
Comment (seaerchin) [inline src/layouts/components/Editable/Editable.tsx:225]: assumed to be cleared with design

**Review by seaerchin — COMMENTED** [review]
1 sanity check wrt a change in `GeneralInfoSection`; for the rest of the changes in `Editable`, just confirming that these changes are cleared by design to be correct also for the rest of the pages

File: src/layouts/components/ContactUs/GeneralInfoSection.tsx (hunk 1 above)
Comment (dcshzj) [reply src/layouts/components/ContactUs/GeneralInfoSection.tsx:39]: Yup, the rule is "the padding should be larger if there is no action button at the end of the card".

File: src/layouts/components/Editable/AddSectionButton.tsx (hunk 2 above)
Comment (dcshzj) [reply src/layouts/components/Editable/AddSectionButton.tsx:23]: Partially yes. Design mentioned that it should be the `lg` size of the `Button` prop in the design system. However, we need to use the Menu button to support a dropdown menu for the homepage. So design will follow-up separately with design system team to synchronise the size values for both the Button and Menu components.

For now, this is okay until design system updates to have the correct sizing, no changes will be needed from our end.

File: src/layouts/components/Editable/Editable.tsx (hunk 3 above)
Comment (dcshzj) [reply src/layouts/components/Editable/Editable.tsx]: Partially yes, was updated in 5ae1776da0d577e004a2653cec54912e666d40b3.

File: src/layouts/components/Editable/Editable.tsx (hunk 4 above)
Comment (dcshzj) [reply src/layouts/components/Editable/Editable.tsx:225]: Yes, the accordion was too big originally.

**Review by seaerchin — APPROVED** [review]

**Pull request merged**